src/main/java/net/minestom/server/MinecraftServer.java

```diff
@@ -202,89 +216,44 @@ public static boolean isStopping() {
+     * @deprecated use {@link Configuration#chunkViewDistance()}
      */
     @Deprecated
-    public static void setChunkViewDistance(int chunkViewDistance) {
-        Check.stateCondition(serverProcess.isAlive(), "You cannot change the chunk view distance after the server has been started.");
-        Check.argCondition(!MathUtils.isBetween(chunkViewDistance, 2, 32),
-                "The chunk view distance must be between 2 and 32");
-        MinecraftServer.chunkViewDistance = chunkViewDistance;
+    public static int getChunkViewDistance() {
+        return configuration().chunkViewDistance();
     }
 
     /**
      * Gets the entity view distance of the server.
      *
      * @return the entity view distance
-     */
-    public static int getEntityViewDistance() {
-        return entityViewDistance;
-    }
-
-    /**
-     * Changes the entity view distance of the server.
-     *
-     * @param entityViewDistance the new entity view distance
-     * @throws IllegalArgumentException if {@code entityViewDistance} is not between 0 and 32
-     * @deprecated should instead be defined with a java property
+     * @deprecated use {@link Configuration#entityViewDistance()}
      */
     @Deprecated
-    public static void setEntityViewDistance(int entityViewDistance) {
-        Check.stateCondition(serverProcess.isAlive(), "You cannot change the entity view distance after the server has been started.");
-        Check.argCondition(!MathUtils.isBetween(entityViewDistance, 0, 32),
-                "The entity view distance must be between 0 and 32");
-        MinecraftServer.entityViewDistance = entityViewDistance;
+    public static int getEntityViewDistance() {
+        return configuration().entityViewDistance();
     }
 
     /**
      * Gets the compression threshold of the server.
      *
      * @return the compression threshold, 0 means that compression is disabled
+     * @deprecated use {@link Configuration#compressionThreshold()}
      */
+    @Deprecated
     public static int getCompressionThreshold() {
-        return compressionThreshold;
-    }
-
-    /**
-     * Changes the compression threshold of the server.
-     * <p>
-     * WARNING: this need to be called before {@link #start(SocketAddress)}.
-     *
-     * @param compressionThreshold the new compression threshold, 0 to disable compression
-     * @throws IllegalStateException if this is called after the server started
-     */
-    public static void setCompressionThreshold(int compressionThreshold) {
-        Check.stateCondition(serverProcess.isAlive(), "The compression threshold cannot be changed after the server has been started.");
-        MinecraftServer.compressionThreshold = compressionThreshold;
+        return configuration().compressionThreshold();
     }
 
     /**
      * Gets if the built in Minestom terminal is enabled.
      *
      * @return true if the terminal is enabled
+     * @deprecated use {@link }
```

Comment:
lol

Reply:
It's definitelly not because I was too lazy to write the annotation and javadoc for everything and just copy-pasted the template and forgot to fill in the detail...